File: src/MSAmanager/app/src/main/java/ms_alum/azrieli_college_of_engineering/com/msamanager/kablni_mishne_activity.java
package ms_alum.azrieli_college_of_engineering.com.msamanager;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Button;

public class kablni_mishne_activity extends AppCompatActivity {

    private Button btn_simple;
    private Button btn_last_transactions;
    private Button btn_more_info;
    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_kablni_mishne_activity);
        //subroutine to connect GUI to program
        btn_simple = (Button)findViewById(R.id.kablni_mishne_pshot);
        btn_last_transactions = (Button)findViewById(R.id.kablni_mishne_ntonim_hodshiim);
        btn_more_info = (Button)findViewById(R.id.kablni_mishne_pirot);

    }
}

File: src/app/src/main/java/com/example/chaosruler/msa_manager/kablni_mishne_activity.kt
package com.example.chaosruler.msa_manager

import android.app.Activity
import android.os.Bundle

class kablni_mishne_activity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        setTheme(themer.style(baseContext))
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_kablni_mishne_activity)
    }
}
